src/debug-generator/debug-generator.c

```diff
@@ -66,6 +66,17 @@ static const struct BreakpointInfo breakpoint_info_table[_BREAKPOINT_TYPE_MAX] =
         { BREAKPOINT_PRE_SWITCH_ROOT,   "pre-switch-root", "breakpoint-pre-switch-root.service", BREAKPOINT_IN_INITRD | BREAKPOINT_DEFAULT },
 };
 
+static bool breakpoint_applies(const BreakpointInfo *info, int log_level) {
```

Comment:
The `log_level` argument is always `LOG_WARNING` ? Why not to use `log_warning()` ?

Reply:
I just wrote the suggested arguments: https://github.com/systemd/systemd/pull/35410#discussion_r1893667581

Yes, let's wait for his feedback.